fcrepo-persistence-ocfl/src/main/java/org/fcrepo/persistence/ocfl/RepositoryInitializer.java

```diff
@@ -54,6 +63,13 @@
     @Inject
     private FedoraToOCFLObjectIndexUtil fedoraToOCFLObjectIndexUtil;
 
+    private static Stream<Triple> REPOSITORY_ROOT_TRIPLES = Stream.of(
```

Comment:
Since repository initialization is a one-time event, we may not want to retain these two static class constants. 
Suggestion: move them into the `initialize()` method as local variables.